native/swift/Sources/wordpress-api/Exports.swift

```diff
@@ -129,3 +131,7 @@ public typealias WpSiteHealthTestsRequestExecutor = WordPressAPIInternal.WpSiteH
 
 // MARK: - WordPress.org
 public typealias WordPressOrgApiClient = WordPressAPIInternal.WordPressOrgApiClient
+
+// MARK: - WordPress.com
+public typealias WPComApiClient = WordPressAPIInternal.UniffiWpComApiClient
```

Comment:
Sorry if I'm repeating something you are already aware of, because I roughly remember you talked about similar things.

I don't think we should expose uniffi clients as a public API. We should always wrap them in a Swift type, like how `WordPressAPI` wraps `UniffiWpApiClient`. One major pitfall I found is that the `WpApiClientDelegate` instance passed to the uniffi API clients is not retained, and we have to manually retain them in the wrapper like `WordPressAPI`.

Reply:
Yep you're right! Addressed in https://github.com/Automattic/wordpress-rs/pull/733/commits/3b950894cee4353b60a7be5feba8bdbd0e281dd9